libkbfs/key_manager.go

```diff
@@ -31,36 +31,36 @@ func NewKeyManagerStandard(config Config) *KeyManagerStandard {
 // GetTLFCryptKeyForEncryption implements the KeyManager interface for
 // KeyManagerStandard.
 func (km *KeyManagerStandard) GetTLFCryptKeyForEncryption(ctx context.Context,
-	md ReadOnlyRootMetadata) (tlfCryptKey TLFCryptKey, err error) {
-	return km.getTLFCryptKeyUsingCurrentDevice(ctx, md,
-		md.LatestKeyGeneration(), false)
+	kmd KeyMetadata) (tlfCryptKey TLFCryptKey, err error) {
+	return km.getTLFCryptKeyUsingCurrentDevice(ctx, kmd,
+		kmd.LatestKeyGeneration(), false)
 }
 
 // GetTLFCryptKeyForMDDecryption implements the KeyManager interface
 // for KeyManagerStandard.
 func (km *KeyManagerStandard) GetTLFCryptKeyForMDDecryption(
-	ctx context.Context, mdToDecrypt, mdWithKeys ReadOnlyRootMetadata) (
+	ctx context.Context, kmdToDecrypt, kmdWithKeys KeyMetadata) (
```

Comment:
It can, but I prefer to keep that implementation detail (the fact that the most recent one is used) as part of `KeyManager`.